src/Math-Matrix/PMSingularValueDecomposition.class.st

```diff
@@ -56,26 +56,26 @@ PMSingularValueDecomposition >> initialize: aMatrix [
 	"Expensive computation"
 	eigenU := symU eigen.
 	eigenV := symV eigen.
-	u := (PMMatrix rows: eigenU vectors) transpose.
-	v := (PMMatrix rows: eigenV vectors) transpose.
+	leftSingularForm := (PMMatrix rows: eigenU vectors) transpose.
+	rightSingularForm := (PMMatrix rows: eigenV vectors) transpose.
 	diag := m < n
 		        ifTrue: [ eigenU values ]
 		        ifFalse: [ eigenV values ].
-	s := PMMatrix rows: m columns: n random: 0.
-	s setDiagonal: diag sqrt
+	sForm := PMMatrix rows: m columns: n random: 0.
+	sForm setDiagonal: diag sqrt
```

Comment:
It looks like this is diagonalised square root. Is that right? If so, perhaps the name could be diagonalisedSquareRoot?